dotCMS/src/main/java/com/dotcms/rest/StringCommaSeparatedList.java

```diff
@@ -0,0 +1,30 @@
+package com.dotcms.rest;
+
+import com.liferay.util.StringPool;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Just a simple cast to bind a list of String comma separated into an array list.
+ * This could be used as:
+ * <code>
+ *     @Path("/ids/{strings}")
+ *     public Response mymethod(@Context HttpServletRequest request,
+ *                              @Context final HttpServletResponse response,
+ *                              final StringCommaSeparatedList idList) {
+ * </code>
+ *
+ * @author jsanca
+ */
+public class StringCommaSeparatedList extends ArrayList<String> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)